view/LoggedInView.java

```diff
@@ -1,2 +1,212 @@
+    private final JTextField currentTemperatureDisplay;
+    private final JLabel currentTemperatureLabel;
+    private final JLabel dailyMaxMinLabel;
+    private final JTextField dailyMaxMinTempsDisplay;
+    private final JTextField futureDataDisplay;
+    private final JLabel futureDataDisplayLabel;
+    private final JLabel precipitationPercentageLabel;
+    private final JTextField precipitationPercentageDisplay;
+    private final JButton searchButton;
+    private final JButton settingsButton;
+    private final JButton weatherIconDisplay;
+    private final JButton logoutButton;
+    private final JLabel bufferField1;
+    private final JLabel bufferField2;
+
+    private final LoggedInViewModel loggedInViewModel;
+
+    // End of variable declaration
+
+    // TODO: ADD ALL USE CASE CONTROLLERS TO INITIALIZER "LoginController controller, "
+
+    public LoggedInView(LoggedInViewModel loggedInViewModel) {
+        /**
+         * TODO: Impliment basic controler classes for the view functionality
+         */
+
+        // this.logiController = controller;
+        this.loggedInViewModel = loggedInViewModel;
+        loggedInViewModel.addPropertyChangedListener(this);
+
+        // Object creation for visual elements
+
+        // JLabels - titles and labels
+        applicationTitle = new JLabel(loggedInViewModel.APPLICATION_TITLE);
+        currentTemperatureLabel = new JLabel(loggedInViewModel.CURRENT_TEMPERATURE_LABEL);
+        dailyMaxMinLabel = new JLabel(loggedInViewModel.DAILY_MAX_MIN_LABEL);
+        futureDataDisplayLabel = new JLabel(loggedInViewModel.FUTURE_DATA_LABEL);
+        precipitationPercentageLabel = new JLabel(loggedInViewModel.PRECIPITATION_CHANCE_LABEL);
+        bufferField1 = new JLabel(loggedInViewModel.BUFFER_EMPTY_TEXT);
+        bufferField2 = new JLabel(loggedInViewModel.BUFFER_EMPTY_TEXT);
+
+        // JTextFields - return areas for weather data and search intake area
+        // Input field
+        citySearchInputField = new JTextField(loggedInViewModel.CITY_SEARCH_DEFAULT_TEXT);
+        //Output fields
+        currentTemperatureDisplay = new JTextField(loggedInViewModel.CURRENT_TEMPERATURE_DEFAULT_TEXT);
+        dailyMaxMinTempsDisplay = new JTextField(loggedInViewModel.DAILY_MAX_MIN_DEFAULT_TEXT);
+        futureDataDisplay = new JTextField(loggedInViewModel.FUTURE_DATA_DEFAULT_TEXT);
+        precipitationPercentageDisplay = new JTextField(loggedInViewModel.PRECIPITATION_CHANCE_DEFAULT_TEXT);
+
+        // JButtons - interaction points for the user and icon display area
+        // Input button - requires listeners
+        createNotificationButton = new JButton(loggedInViewModel.CREATE_NOTIFICATION_BUTTON_LABEL);
+        searchButton = new JButton(loggedInViewModel.CITY_SEARCH_BUTTON_LABEL);
+        settingsButton = new JButton(loggedInViewModel.SETTING_BUTTON_IMG);
+        logoutButton = new JButton(loggedInViewModel.LOGOUT_BUTTON_TEXT);
+        // Output button - no functionality on click
```

Comment:
If you remake the LoggedinViewModel text labels to be _static_, you do not have to assign these values in the constructor. You can just do it directly above when declaring (e.g. `private final JLabel applicationTitle = LoggedInViewModel.APPLICATION_TITLE`)